osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/scheduling/OsgpScheduler.java

```diff
@@ -130,24 +149,47 @@ public Trigger createJobTrigger(final JobDetail jobDetail, final String cronExpr
      * @param cronExpression
      *            The input for the trigger, a Quartz CRON expression like
      *            {@code 0 0/1 * * * ?} for example.
+     * @param timeZone
+     *            A Timezone for the CRON-expression.
      *
      * @throws SchedulerException
      *             In case adding or scheduling of the job fails.
      */
-    public void createAndScheduleJob(final Class<? extends Job> jobClass, final String cronExpression)
+    public void createAndScheduleJob(final Class<? extends Job> jobClass, final String cronExpression,
+            final TimeZone timeZone)
             throws SchedulerException {
 
         LOGGER.info("Scheduling job: {} using Quartz cron expression: {}", jobClass.getSimpleName(), cronExpression);
 
         // Create job and trigger.
         final JobDetail jobDetail = this.createJobDetail(jobClass);
-        final Trigger trigger = this.createJobTrigger(jobDetail, cronExpression);
+        final Trigger trigger = this.createJobTrigger(jobDetail, cronExpression, timeZone);
 
         // Add and schedule for trigger.
         this.quartzScheduler.addJob(jobDetail, true);
         this.quartzScheduler.scheduleJob(jobDetail, new HashSet<>(Arrays.asList(trigger)), true);
     }
 
+    /**
+     * Convenience method for creating a job and trigger, then adding and
+     * scheduling the job using the trigger. This method uses
+     * {@link OsgpScheduler#createJobDetail(Class)} and
+     * {@link OsgpScheduler#createJobTrigger(JobDetail, String)}.
+     *
+     * @param jobClass
+     *            The class which defines the actions of the scheduled job.
+     * @param cronExpression
+     *            The input for the trigger, a Quartz CRON expression like
+     *            {@code 0 0/1 * * * ?} for example.
+     *
+     * @throws SchedulerException
+     *             In case adding or scheduling of the job fails.
+     */
+    public void createAndScheduleJob(final Class<? extends Job> jobClass, final String cronExpression)
+            throws SchedulerException {
+        createAndScheduleJob(jobClass, cronExpression, TimeZone.getDefault());
```

Comment:
Same as previous comment (and additionally consider using a constant for the UTC TimeZone):
```suggestion
        this.createAndScheduleJob(jobClass, cronExpression, TimeZone.getTimeZone("UTC"));
```
Consider defining a constant for the UTC TimeZone.

Reply:
See comment above

Reply:
Changed